framework/rbac/DbManager.php

```diff
@@ -753,6 +754,24 @@ public function hasChild($parent, $child)
     /**
      * @inheritdoc
      */
+    public function getParents($name)
```

Comment:
This would be inconsistent with `getChildren()` and require its refactoring too.